lib/Runtime/Library/ForInObjectEnumerator.h

```diff
@@ -21,7 +21,7 @@ namespace Js
         } *shadowData;
 
         // States
-        bool canUseJitFastPath;
+        bool canUseJitFastPath; // used by GenerateFastBrBReturn and GenerateFastInlineHasOwnProperty
```

Comment:
Would be good to document what this being true means here now that there are other use of this boolean.